Exiled.Loader/ConfigManager.cs

```diff
@@ -40,9 +40,15 @@ public static class ConfigManager
                 Dictionary<string, object> rawDeserializedConfigs = Loader.Deserializer.Deserialize<Dictionary<string, object>>(rawConfigs) ?? DictionaryPool<string, object>.Pool.Get();
                 SortedDictionary<string, IConfig> deserializedConfigs = new(StringComparer.Ordinal);
 
+                // Only allow unique keys to be loaded. Any duplicates in the plugin's config will be skipped.
                 foreach (IPlugin<IConfig> plugin in Loader.Plugins)
                 {
-                    deserializedConfigs.Add(plugin.Prefix, plugin.LoadConfig(rawDeserializedConfigs));
+                    if (!deserializedConfigs.ContainsKey(plugin.Prefix))
+                    {
+                        deserializedConfigs.Add(plugin.Prefix, plugin.LoadConfig(rawDeserializedConfigs));
+                    } else {
```

Comment:
![immagine](https://github.com/Exiled-Team/EXILED/assets/60253860/57f3943a-bb21-4ebc-8af6-61af09b82139)

This bracing style won't be accepted.

Reply:
Updated Bracing Style.